engine/src/test/scala/engine/db/slick/SlickDataAccessSpec.scala

```diff
@@ -6,7 +6,7 @@ import java.util.UUID
 import better.files._
 import com.google.api.client.util.ExponentialBackOff
 import com.typesafe.config.ConfigFactory
-import cromwell.CromwellSpec.IntegrationTest
+import cromwell.CromwellSpec.{DbmsTest, IntegrationTest}
```

Comment:
There's still a test tagged as `IntegrationTest`. Delete this import and intellij/sbt shall reveal the line.